core/model/modx/processors/system/config_check.inc.php

```diff
@@ -91,7 +91,11 @@ function getResourceBypass(modX &$modx,$criteria) {
         curl_setopt($curl, CURLOPT_VERBOSE, false);
         /* follow rewrites, e.g. http to https rewrites */
         /* if a sites rewrites the checkurl to 200 page, this method fails */
-        curl_setopt($curl, CURLOPT_FOLLOWLOCATION, true);
+        $safeMode = @ini_get('safe_mode');
```

Comment:
Why do you use error suppression? Using @ turns off all optimization in the whole file. Better use extra checks and avoid using @.